project/Build.scala

```diff
@@ -68,9 +68,10 @@ object Build extends AutoPlugin {
       val benchRscNativeTypecheck = "benchCliRscNativeTypecheck"
       private val rscTyper = List("ColdRscTypecheck", "HotRscTypecheck")
       private val scalaTyper = List("ColdScalacTypecheck", "HotScalacTypecheck")
+      private val scalaNamer = List("ColdScalacNamer", "HotScalacNamer")
       private val scalaCompile = List("ColdScalacCompile", "HotScalacCompile")
       private val javaCompile = List("ColdJavacCompile", "HotScalacCompile")
-      private val all = rscTyper ++ scalaTyper ++ scalaCompile ++ javaCompile
+      private val all = rscTyper ++ scalaNamer ++ scalaTyper ++ scalaCompile ++ javaCompile
```

Comment:
As described in the comment above, please add `rscScheduler` here and rerun `sbt bench`.

Reply:
I won't have time to do this in the next couple of days, but I am happy to run this benchmark too. Can you share your code for rscScheduler? Any issue with going to 2.12.4 across the board, so we don't have an explosion of numbers?

Reply:
I've refactored our benchmarking infrastructure to accommodate both Scala 2.11 and Scala 2.12. Please see https://github.com/twitter/reasonable-scala/pull/27.